extension-base/ctpAzureWrapper.js

```diff
@@ -0,0 +1,77 @@
+// Azure
+
+module.exports = {
+  ctpAzureWrapper: function(fn) {
```

Comment:
Personally I would call it adapter not wrapper but that's just cause I think wrapper can be anything while adapter is something plugging into something else. Obviously subjective.